File: Llama 3.1 70B/C++/2-3.c
// File Read and Write Operations Program

#include <iostream>
#include <fstream>
#include <string>
using namespace std;

void readFile(const string& filename) {
    ifstream file(filename);
    if (file.is_open()) {
        string line;
        while (getline(file, line)) {
            cout << line << endl;
        }
        file.close();
    } else {
        cout << "Unable to open file." << endl;
    }
}

void writeFile(const string& filename, const string& content) {
    ofstream file(filename);
    if (file.is_open()) {
        file << content;
        file.close();
    } else {
        cout << "Unable to open file." << endl;
    }
}

int main() {
    string inputFilename, outputFilename;

    cout << "File Read and Write Operations Program" << endl;
    cout << "Enter the input filename: ";
    cin >> inputFilename;

    cout << "Enter the output filename: ";
    cin >> outputFilename;

    string content;
    readFile(inputFilename);
    cout << "Enter the content to write to the output file: ";
    cin.ignore();
    getline(cin, content);
    writeFile(outputFilename, content);

    return 0;
}

File: Llama 3.1 70B/Python/2-3.py
# File Read and Write Operations Program

def read_file(filename):
    """Returns the content of a file."""
    try:
        with open(filename, 'r') as file:
            return file.read()
    except FileNotFoundError:
        print("File not found!")
        return None

def write_file(filename, content):
    """Writes content to a file."""
    with open(filename, 'w') as file:
        file.write(content)

def main():
    print("File Read and Write Operations Program")
    input_filename = input("Enter the input filename: ")
    output_filename = input("Enter the output filename: ")

    content = read_file(input_filename)
    if content:
        write_file(output_filename, content)
        print("Content written to", output_filename)

if __name__ == "__main__":
    main()

